ios/FluentUI/Vnext/Button/ButtonTokens.swift

```diff
@@ -15,6 +15,8 @@ import SwiftUI
     case accentFloating
     /// For use with small and large sizes only
     case subtleFloating
+
+    var isFloatingStyle: Bool { return self == .accentFloating || self == .subtleFloating }
```

Comment:
style nit: can you add a new line after the opening curly brace and another one before the closing curly brace?